SimplyPluralApiBodies.yml

```diff
@@ -72,69 +153,237 @@ components:
+          type: string
+        desc:
+          description:
+            $ref: "#/components/schemas/chatCategory/properties/desc/description"
+          type: string
+        channels:
+          description:
+            $ref: "#/components/schemas/chatCategory/properties/desc/description"
+          type: array
+          items:
+            type: string
+        uid:
+          description:
+            $ref: "./SimplyPluralApiCommons.yml#/components/schemas/commons/properties/uid/description"
+          type: string
+        lastOperationTime:
+          description:
+            $ref: "./SimplyPluralApiCommons.yml#/components/schemas/commons/properties/lastOperationTime/description"
+          type: long
+
+    chatChannel:
+      type: object
+      properties:
+        name:
+          description: Channel name
+          type: string
+        desc:
+          description: Description of the channel
+          type: string
+
+    chatChannelContainer:
+      type: object
+      properties:
+        exists:
+          type: boolean
+        id:
+          description: Message Id
+          type: string
+        content:
+          description: The message itself
+          $ref: "#/components/schemas/chatChannelInfo"
+
+    chatChannelInfo:
+      type: object
+      properties:
+        name:
+          description:
+            $ref: "#/components/schemas/chatChannel/properties/name/description"
+          type: string
+        desc:
+          description:
+            $ref: "#/components/schemas/chatChannel/properties/desc/description"
+          type: string
+        uid:
+          description:
+            $ref: "./SimplyPluralApiCommons.yml#/components/schemas/commons/properties/uid/description"
+          type: string
+        lastOperationTime:
+          description:
+            $ref: "./SimplyPluralApiCommons.yml#/components/schemas/commons/properties/lastOperationTime/description"
+          type: long
+
+    chatMessage:
       type: object
       properties:
         message:
           description: Message
           type: string
         channel:
-          description: ID of the channel this message should go to
+          description: Id of the channel this message should go to
           type: string
         writer:
-          description: ID of the member that wrote this message
+          description: Id of the member that wrote this message
           type: string
         writtenAt:
           description:
             $ref: "./SimplyPluralApiCommons.yml#/components/schemas/commons/properties/timestamp/description"
           type: integer
         replyTo:
-          description: "Message ID this message is in reply to"
+          description: Message Id this message is in reply to
           type: string
 
-    chatMessagePatch:
+    chatMessageContainer:
+      type: object
+      properties:
+        exists:
+          type: boolean
+        id:
+          description: Message Id this message is in reply to
```

Comment:
that's not correct, the id of the response is the id of the actual message retrieved. "replyTo" is the only field that references a replied message